tests/PatternBuilder.test.ts

```diff
@@ -10,16 +10,39 @@ describe('PatternBuilder', () => {
     PB.subgroup,
     PB.maybe,
     PB.then(pipe(PB.anything, PB.anyNumber({ greedy: true }))),
+    PB.then(pipe(PB.anything, PB.anyNumber({ greedy: false }))),
```

Comment:
The changes in this file are just to increase test coverage for this file.  It's almost 100%!